### PR TITLE
improve(gas-estimator): Add Boba as legacy network

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

Boba requires legacy transactions to be sent.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [X]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/financial-templates-lib/src/helpers/GasEstimator.ts [hunk 1]
```diff
@@ -182,7 +192,22 @@ export class GasEstimator {
     const url = MAPPING_BY_NETWORK[_networkId].url;
     const backupUrl = MAPPING_BY_NETWORK[_networkId].backupUrl;
 
-    if (!url) throw new Error(`Missing URL for network ID ${_networkId}`);
+    if (!url) {
+      // If no URL specified, use default.
+      if (MAPPING_BY_NETWORK[_networkId].defaultFastPriceGwei)
+        return {
+          gasPrice: MAPPING_BY_NETWORK[_networkId].defaultFastPriceGwei,
+        } as LegacyGasData;
```
Comment (mrice32) [inline packages/financial-templates-lib/src/helpers/GasEstimator.ts:200]: odd that you have to cast this

File: packages/financial-templates-lib/src/helpers/GasEstimator.ts [hunk 2]
```diff
@@ -182,7 +192,22 @@ export class GasEstimator {
...
+      else if (
+        MAPPING_BY_NETWORK[_networkId].defaultMaxFeePerGasGwei &&
+        MAPPING_BY_NETWORK[_networkId].defaultMaxPriorityFeePerGasGwei
+      )
+        return {
+          maxFeePerGas: this.defaultMaxFeePerGasGwei,
+          maxPriorityFeePerGas: this.defaultMaxPriorityFeePerGasGwei,
+        } as LondonGasData;
```
Comment (mrice32) [inline packages/financial-templates-lib/src/helpers/GasEstimator.ts:208]: Same here.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by chrismaree — APPROVED** [review]
you tested this in a local node instance, right?

File: packages/financial-templates-lib/src/helpers/GasEstimator.ts (hunk 1 above)
Comment (nicholaspai) [reply packages/financial-templates-lib/src/helpers/GasEstimator.ts:200]: ah yeah i don't will fix

File: packages/common/src/TransactionUtils.ts [hunk 3]
```diff
@@ -1,6 +1,5 @@
 import util from "util";
 import minimist from "minimist";
-import ynatm from "@umaprotocol/ynatm";
```
Comment (chrismaree) [inline packages/common/src/TransactionUtils.ts:3]: you can probs remove this from the associated `package.json` for `common`, no?

Comment (chrismaree) [reply packages/common/src/TransactionUtils.ts:3]: it actually looks like `ynatm` is in a few other packages where it probs does not need to be. I dont see a point in it being in liquidator or trader. I think you can remove it from both of those places as well, no?

Comment (nicholaspai) [reply packages/common/src/TransactionUtils.ts:3]: done

**Pull request merged**